charts/invenio/templates/web-deployment.yaml

```diff
@@ -73,28 +73,9 @@ spec:
               name: {{ .valueFrom.secretKeyRef.name }}
               key: {{ .valueFrom.secretKeyRef.key }}
         {{- end }}
-        readinessProbe:
-          exec:
-            command:
-              - /bin/bash
-              - -c
-              - "uwsgi_curl -X HEAD -H 'Host: {{ include "invenio.hostname" $ }}' $(hostname):5000 /ping"
-          failureThreshold: {{ .Values.web.readinessProbe.failureThreshold }}
-          initialDelaySeconds: {{ .Values.web.readinessProbe.initialDelaySeconds }}
-          periodSeconds: {{ .Values.web.readinessProbe.periodSeconds }}
-          successThreshold: {{ .Values.web.readinessProbe.successThreshold }}
-          timeoutSeconds: {{ .Values.web.readinessProbe.timeoutSeconds }}
-        startupProbe:
-          exec:
-            command:
-              - /bin/bash
-              - -c
-              - "uwsgi_curl -X HEAD -H 'Host: {{ include "invenio.hostname" $ }}' $(hostname):5000 /ping"
-          failureThreshold: {{ .Values.web.startupProbe.failureThreshold }}
-          initialDelaySeconds: {{ .Values.web.startupProbe.initialDelaySeconds }}
-          periodSeconds: {{ .Values.web.startupProbe.periodSeconds }}
-          successThreshold: {{ .Values.web.startupProbe.successThreshold }}
-          timeoutSeconds: {{ .Values.web.startupProbe.timeoutSeconds }}
+        livenessProbe: {{- tpl (.Values.web.livenessProbe | toYaml) $ | nindent 10 }}
```

Comment:
Would it be a problem to have it empty?

Reply:
Seems to work fine. Setting it to `{}` generates `{}` in the manifest and setting it to `null` generates `null` in the manifest.